src/screens/auth/LoginScreen.tsx

```diff
@@ -54,12 +57,13 @@ export default function LoginScreen() {
 
   return (
     <LoginContainer>
-      {/* logo placeholder */}
-      <LogoContainer>
-        <View style={{ backgroundColor: 'black', width: 50, height: 59.29 }}>
-          <Text style={{ color: 'white' }}>{`\n  Logo`}</Text>
-        </View>
-      </LogoContainer>
+      <BackButtonContainer>
+        <RootNavBackButton onPress={() => navigation.navigate('Start')}>
+          <Text>
+            <Icon name="left" size={14} color="#525454" /> Back
```

Comment:
Use the darkGray color from our global `Colors.ts` file. You can do this by setting the color as `${Colors.darkGray}`.